src/coord/src/coord.rs

```diff
@@ -914,13 +883,7 @@ impl Coordinator {
                     }
                 }
             }
-            DataflowResponse::Compute(ComputeResponse::FrontierUppers(updates), instance) => {
-                assert_eq!(instance, DEFAULT_COMPUTE_INSTANCE_ID);
-                for (name, changes) in updates {
-                    self.update_upper(&name, changes);
-                }
-                self.maintenance().await;
-            }
+            DataflowResponse::Compute(ComputeResponse::FrontierUppers(_updates), _instance) => {}
```

Comment:
Remove variant, potentially. Here for exhaustiveness, but .. morally there is nothing for the coordinator to do. Could prompt it to reconsider queries against certain indexes.

Reply:
From the perspective of command reconciliation, removing it would be one less thing to take care of!